### PR TITLE
Add a link to drive-rs

### PR DESCRIPTION
**Pull request by tdittr**
This might need to be discussed in the next meeting.

### CODE REVIEW

**Comment by BartMassey** [comment]
Yes, good discussion for next meeting. `drive-rs` looks cool. It seems unnecessary to keep driver lists two separate places. For myself, I would be fine with whichever one wants to pledge to keep things up to date being the sole source of truth… It doesn't matter much to me where folks find drivers, as long as they can find drivers.

File: README.md [hunk 1]
```diff
@@ -759,6 +759,8 @@ devices that go beyond what is available in [`embedded-hal`]:
 
 ## Driver crates
 
+For more drivers also refer to [drive-rs](https://tweedegolf.github.io/drive-rs/).
```
Comment (tdittr) [inline README.md:762]:
```suggestion
You can also find many of these drivers [here](https://tweedegolf.github.io/drive-rs/); please consider adding your driver there (https://github.com/tweedegolf/drive-rs) while we work to port everything across.
```
As suggested by @adamgreig  in the meeting.

File: README.md [hunk 2]
```diff
@@ -759,6 +759,8 @@ devices that go beyond what is available in [`embedded-hal`]:
 
 ## Driver crates
 
+You can also find many of these drivers [here](https://tweedegolf.github.io/drive-rs/); please consider adding your driver there (https://github.com/tweedegolf/drive-rs) while we work to port everything across.
```
Comment (rursprung) [inline README.md:762]:
```suggestion
You can also find many of these drivers [on drive-rs](https://tweedegolf.github.io/drive-rs/); please consider adding your driver there ([in the GitHub repo](https://github.com/tweedegolf/drive-rs)) while we work to port everything across.
```

as per [standard accessibility guidelines](https://webaim.org/techniques/hypertext/link_text#uninformative) you shouldn't name links "here" (e.g. important for people with screen readers)

**Review by rursprung — CHANGES_REQUESTED** [review]
drive-rs looks very nice!
just a small nit about the wording of the link text

File: README.md (hunk 2 above)
Comment (tdittr) [reply README.md:762]: oh, good to know, thanks :) also reads much better!

**Review by adamgreig — APPROVED** [review]

**Pull request merged**